src/core/searchUtils.js

```diff
@@ -108,10 +108,22 @@ export function loadByCategoryIfNeeded(
   { store: { dispatch, getState }, location, params }
 ) {
   const state = getState();
+
+  // HACK: This loads Firefox personas (lightweight themes) for Android
+  // until https://github.com/mozilla/addons-frontend/issues/1723#issuecomment-278793546
+  // is more fully addressed.
+  // Essentially: right now there are no categories for the combo
+  // of "Android" + "Themes" but Firefox lightweight themes will work fine
+  // on mobile so we request them.
+  // Obviously we need to fix this on the API end so our requests aren't
+  // overridden, but for now this will work.
+  const clientApp = params.application === 'android' ?
```

Comment:
Yeah actually, I can do it in there for sure. I think I put it in two places because it compares the filters sent to callApi and those in state... but putting in it `search` is better I think.